### PR TITLE
Add timing lookup

### PR DESCRIPTION
**Pull request by clark2668**
This adds a small helper function to `araproc.analysis.standard_reco` to lookup the arrival time at an antenna from the ray trace correlator table.

### CODE REVIEW

**Comment by abigailbishop** [comment]
I have no comments, this looks great! Thanks Brian!

File: examples/run_example.py [hunk 1]
```diff
@@ -85,11 +85,21 @@
...
+        print(f"The Average SNR is {avg_snr:.1f}")
  
         # run our standard suite of reconstructions
         reco_results = reco.do_standard_reco(wave_bundle)
 
+        # here's how we can lookup arrival times given a reconstructed direction
+        arrival_time = reco.lookup_arrival_time(channel = 0, 
+                                                theta = reco_results["pulser_v"]["theta"], 
+                                                phi = reco_results["pulser_v"]["phi"],
+                                                which_distance="distant",
```
Comment (abigailbishop) [inline examples/run_example.py:98]: I feel like this is a silly question on my side but can you remind me why we're paying pulser reconstructions results with `which_distance="distant"`?

Comment (clark2668) [reply examples/run_example.py:98]: Because.... I made a mistake. Fixed now haha! Thank you for catching this!

**Comment by clark2668** [comment]
Thanks @abigailbishop , you caught a bug in my application of the function in the example script! Fixed now, and going to merge.

**Pull request merged**